nautobot_chatops_ipfabric/worker.py

```diff
@@ -513,10 +532,18 @@ def pathlookup(
         ]
     )
 
-    # only supported in IP Fabric OS version 4.0+
+    # diagrams for 4.0 - 4.2 are not supported due to attribute changes in 4.3+
     try:
-        if ipfabric_api.validate_version(ge, 4.0):
-            raw_png = ipfabric_api.get_pathlookup(src_ip, dst_ip, src_port, dst_port, protocol, snapshot_id)
```

Comment:
are we removing this custom method `get_pathlookup`?

Reply:
yes, good idea was going to leave and just remove the module when all migrated but makes sense to do it iteratively.